mesh/pmesh.cpp

```diff
@@ -4647,18 +4647,18 @@ void ParMesh::Print(std::ostream &out) const
          const NCMesh::NCList& sfaces =
             (Dim == 3) ? pncmesh->GetSharedFaces() : pncmesh->GetSharedEdges();
          const int nfaces = GetNumFaces();
-         for (int i = 0; i < sfaces.conforming.Size(); i++)
+         for (i = 0; i < sfaces.conforming.Size(); i++)
```

Comment:
I would prefer `i` to be declared in the loop.

Reply:
As would I. Done.